definitions/divorce/json/CaseEventToFields/CaseEventToFields.json

```diff
@@ -1272,7 +1272,7 @@
     "PageID": "SolStatementOfTruth",
     "PageLabel": "Statement of truth and reconciliation",
     "PageDisplayOrder": 1,
-    "FieldShowCondition": "D8StatementOfTruth=\"Banana\"",
```

Comment:
I'm guessing the idea was not to show this label. ticket requests this to be shown?

Reply:
actually, I don't care

Reply:
This one isn't required.